docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Migrating Airflow to Dagster"
+description: guide for doing a lift and shift migration of airflow to dagster
+---
+
+# Migrating Airflow to Dagster
+
+Dagster can convert your Airflow DAGs into Dagster jobs, making it possible to a lift and shift migration off of Airflow onto Dagster without any rewriting.
+
+This guide will walk you through the steps of doing this migration.
+
+<Note>
+  Looking for example of an Airflow to Dagster migration? Check out the{" "}
+  <a href="https://github.com/dagster-io/dagster-airflow-migration-example">
+    dagster-airflow migration example repo on github
+  </a>
+  !
+</Note>
+
+## Prerequisites
+
+To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+
+```bash
+pip install dagster dagster-airflow
+```
+
+---
+
+## Step 1: Create a new repository using <PyObject module="dagster_airflow" object="make_dagster_repo_from_airflow_dags_path" />
```

Comment:
Headings shouldn't have extra formatting (bold, code, italics, etc) or components. It messes with how anchor links work - can you take the component out?